python/ray/util/collective/util.py

```diff
@@ -56,15 +56,26 @@ def __init__(self):
         self.rank = -1
         self.backend = None
         self.gloo_timeout = 30000
+        self.group_metadata = None
 
-    def set_info(self, ids, world_size, rank, backend, gloo_timeout):
+    def set_info(
+        self, ids, world_size, rank, backend, gloo_timeout, group_metadata=None
+    ):
         """Store collective information."""
         self.ids = ids
         self.world_size = world_size
         self.rank = rank
         self.backend = backend
         self.gloo_timeout = gloo_timeout
+        self.group_metadata = group_metadata
 
     def get_info(self):
         """Get previously stored collective information."""
-        return self.ids, self.world_size, self.rank, self.backend, self.gloo_timeout
+        return (
```

Comment:
The function signature has been updated. We need to check every place where this function is called.

https://github.com/ray-project/ray/blob/a0f71ccd6a98f3062ea3820f10dd9a91a01f657d/python/ray/util/collective/collective.py#L699-L701